indico/modules/rb_new/client/js/common/timeline/DateNavigator.jsx

```diff
@@ -23,6 +23,7 @@ import {Translate} from 'indico/react/i18n';
 import {serializeDate, toMoment} from 'indico/utils/date';
 import {isDateWithinRange} from '../../util';
 
+import './DateNavigator.module.scss';
```

Comment:
not sure why eslint isn't detecting it, but there should be two empty lines after the last import